src/SprykerSdk/Spryk/Model/Spryk/Builder/Transfer/TransferSpryk.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Builder\Transfer;
+
+use DOMDocument;
+use SimpleXMLElement;
+use SprykerSdk\Spryk\Exception\XmlException;
+use SprykerSdk\Spryk\Model\Spryk\Builder\SprykBuilderInterface;
+use SprykerSdk\Spryk\Model\Spryk\Definition\SprykDefinitionInterface;
+use SprykerSdk\Spryk\Style\SprykStyleInterface;
+use Zend\Filter\FilterChain;
```

Comment:
we need adapters I think

Reply:
I asked René, he said that it's ok to have it like this.